src/graph/ShowExecutor.cpp

```diff
@@ -82,18 +82,54 @@ void ShowExecutor::showHosts() {
             return;
         }
 
-        auto retShowHosts = std::move(resp).value();
+        auto hostItems = std::move(resp).value();
         std::vector<cpp2::RowValue> rows;
         std::vector<std::string> header{"Ip", "Port", "Status"};
         resp_ = std::make_unique<cpp2::ExecutionResponse>();
+
+        header.emplace_back("Ip");
```

Comment:
Use 

`header{"Ip", "Port", "Status", "Leader Count", "Leader Distribution", "Partition Distribution"}` 

will be better ?

Reply:
good suggestion